plugin.video.fosdem/addon.xml

```diff
@@ -1,5 +1,5 @@
 <?xml version="1.0" encoding="UTF-8" standalone="yes"?>
-<addon id="plugin.video.fosdem" name="FOSDEM Videos" version="0.0.6" provider-name="Jelle van der Waa">
+<addon id="plugin.video.fosdem" name="FOSDEM Videos" version="0.0.7" provider-name="Jelle van der Waa">
```

Comment:
Is it correct that this is version 0.0.7 and the Krypton one is 0.0.8? That won't work. Most common is to version the Matrix version with "0.0.8+matrix.1" if the Krypton version is 0.0.8.